src/payment/strategies/klarna/klarna-credit.ts

```diff
@@ -27,3 +27,26 @@ export interface KlarnaAuthorizationResponse {
         invalid_fields: string[];
     };
 }
+
+export interface KlarnaUpdateSessionParams {
+    billing_address: {
+        street_address: string,
+        city: string,
+        country: string,
+        given_name: string,
+        family_name: string,
+        postal_code: string,
+        region: string,
+        email?: string,
```

Comment:
🍹 You could create a separate type calle `KlarnaAddress` and use this type to define `billing_address` and `shipping_address`